fastify.js

```diff
@@ -874,13 +884,18 @@ function build (options) {
     // here, let's print out as much info as
     // we can
     req.id = genReqId(req)
-    req.log = res.log = log.child({ reqId: req.id })
+
+    var logger = log.child({ reqId: req.id })
+    if (optAddToNodeReq) {
+      req.log = res.log = logger
+    }
```

Comment:
We might want to add ip here as well.
Would that be useful?

Reply:
Are you thinking about something like this?
```javascript 
    if (optAddToNodeReq) {
      req.ip = req.connection.remoteAddress
      req.log = res.log = logger
    }
```